nengo/builder/connection.py

```diff
@@ -80,7 +80,11 @@ def build_decoders(model, conn, rng, transform):
         targets = multiply(targets, transform.T)
 
     try:
-        wrapped_solver = model.decoder_cache.wrap_solver(solve_for_decoders)
+        if model.seeded[conn]:
```

Comment:
The relevant code is in `builder/network.py`. A network sets `seeded` for all it's sub-objects (including networks). Thus, a sub-network of a seeded network will also be seeded. It's analogous to how we set the network seeds.